float32 lat
float32 act_lat
float32 lon
float32 act_lon
float32 elevation
float32 act_elevation